### PR TITLE
test: add render test to missed components

### PR DESCRIPTION
**Pull request by peoray**
# Related issue
#620 

# Scope of work
- Added render to components that doesn't have them

# Checklist

- [ ] I followed [composition rules](https://docs.storefrontui.io/contributing/coding-guidelines.html#component-rules) for my component
- [ ] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: packages/vue/src/components/organisms/SfContentPages/_internal/SfContentCategory.vue [hunk 1]
```diff
@@ -1,18 +1,18 @@
 <template>
-  <div>
+  <div class="sf-content-category">
     <slot />
   </div>
 </template>
 <script>
 export default {
-  name: "SfContentCategory",
+  name: 'SfContentCategory',
```
Comment (aniamusial) [inline packages/vue/src/components/organisms/SfContentPages/_internal/SfContentCategory.vue]: Please check if you don't have some kind of local linter enabled - it overwrites the project one. We are using double `" "`, so please fix that :)

File: packages/vue/src/components/organisms/SfFooter/_internal/SfFooterColumn.spec.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import "./matchMedia.mock";
```
Comment (aniamusial) [inline packages/vue/src/components/organisms/SfFooter/_internal/SfFooterColumn.spec.js]: Why is this needed? It causes 'Cannot find module' error

File: packages/vue/src/components/organisms/SfTabs/_internal/SfTab.spec.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import { shallowMount } from "@vue/test-utils";
+import SfTab from "./SfTab.vue";
+describe("SfTab.vue", () => {
+  it("renders a component", () => {
+    const component = shallowMount(SfTab);
+    expect(component.contains(".sf-tabs__title")).toBe(true);
```
Comment (aniamusial) [inline packages/vue/src/components/organisms/SfTabs/_internal/SfTab.spec.js]: This won't work too good - what about `sf-tabs__content` ? You can solve it two ways:
1)  Add 'sf-tabs__tab' class to the wrapper component
2) ` mount` instead of  `shallowMount` to check if it contains `Fragment`: https://vue-test-utils.vuejs.org/api/wrapper/contains.html

File: packages/vue/src/components/organisms/SfFooter/_internal/SfFooterColumn.spec.js (hunk 2 above)
Comment (peoray) [reply packages/vue/src/components/organisms/SfFooter/_internal/SfFooterColumn.spec.js]: I didn't add this there

File: packages/vue/src/components/organisms/SfTabs/_internal/SfTab.spec.js (hunk 3 above)
Comment (peoray) [reply packages/vue/src/components/organisms/SfTabs/_internal/SfTab.spec.js]: Can you throw more light on how to check if it contains Fragment? The docs didn't say much. 
Is this right?
```
import { mount } from '@vue/test-utils'
import SfTab from "./SfTab.vue";
import Fragment from './SfTab.vue'

describe("SfTab.vue", () => {
  it("renders a component", () => {
    const component = mount(SfTab);
    expect(wrapper.contains(Fragment)).toBe(true)
  });
});
```

File: packages/vue/src/components/organisms/SfFooter/_internal/SfFooterColumn.spec.js (hunk 2 above)
Comment (aniamusial) [reply packages/vue/src/components/organisms/SfFooter/_internal/SfFooterColumn.spec.js]: Great :)

File: packages/vue/src/components/organisms/SfTabs/_internal/SfTab.spec.js (hunk 3 above)
Comment (aniamusial) [reply packages/vue/src/components/organisms/SfTabs/_internal/SfTab.spec.js]: component is the wrapper :)This should work fine: 

`import { mount } from '@vue/test-utils'
import SfTab from "./SfTab.vue";
import Fragment from './SfTab.vue'

describe("SfTab.vue", () => {
  it("renders a component", () => {
    const component = mount(SfTab);
    expect(wrapper.contains(Fragment)).toBe(true)
  });
});`

Comment (aniamusial) [reply packages/vue/src/components/organisms/SfTabs/_internal/SfTab.spec.js]: or simply `<Fragment class="sf-tabs__tab">` and check if it renders class

**Comment by aniamusial** [comment]
@pspaczek please have a final look

**Comment by pspaczek** [comment]
thanks, @peoray for your PR 💚

**Comment by peoray** [comment]
Thanks everyone for the help, onto the next one :)
